test/extended/setup.sh

```diff
@@ -22,15 +22,11 @@ function os::test::extended::focus {
 #		be done in other contexts.
 function os::test::extended::setup () {
 	# build binaries
-	if [[ -z "$(os::build::find-binary ginkgo)" ]]; then
-		hack/build-go.sh vendor/github.com/onsi/ginkgo/ginkgo
-	fi
-	if [[ -z "$(os::build::find-binary extended.test)" ]]; then
-		hack/build-go.sh test/extended/extended.test
-	fi
-	if [[ -z "$(os::build::find-binary openshift)" ]]; then
-		hack/build-go.sh
-	fi
+	os::util::ensure::built_binary_exists 'ginkgo' 'vendor/github.com/onsi/ginkgo/ginkgo'
+	os::util::ensure::built_binary_exists 'extended.test' 'test/extended/extended.test'
+	os::util::ensure::built_binary_exists 'openshift'
+	os::util::ensure::built_binary_exists 'oadm'
```

Comment:
No, but since the "cost" of this is basically 0 and we're allowing someone to walk into this test with their own environment I wanted to make sure they didn't walk in with a broken one.